packages/core/package.json

```diff
@@ -18,7 +19,8 @@
     }
   ],
   "dependencies": {
-    "reflect-metadata": "^0.1.13"
+    "reflect-metadata": "^0.1.13",
+    "@monument/collections": "^0.15.0"
```

Comment:
This will create circular package dependencies. Since this moment, collections and core are separate packages. And collections should only depend on core.

Reply:
It will be removed during further steps👌

Right now it's simply to ensure BC and proceed with ejecting next packages